browser/ui/ai_chat/utils.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_AI_CHAT_UTILS_H_
+#define BRAVE_BROWSER_UI_AI_CHAT_UTILS_H_
+
+namespace content {
+class WebContents;
+}
+
+namespace ai_chat {
+
+void OpenLeo(content::WebContents* web_contents);
```

Comment:
Let's avoid the brand name in the code, since this is subject to change. Let's keep it `AIChat`. In this case we want to open a conversation associated with a Tab, so perhaps the function name should be `OpenAIChatForTab`?

Reply:
Renamed to `OpenAIChatForTab` in https://github.com/brave/brave-core/pull/26082/commits/ab4ea07f080379399d945e2357de36394ecf6fc0.